uid2_client/refresh_keys_util.py

```diff
@@ -0,0 +1,70 @@
+import datetime as dt
+import json
+from datetime import timezone
+
+from .keys import EncryptionKey, EncryptionKeysCollection
+from .request_response_util import *
+from .identity_scope import IdentityScope
+
+
+def _make_dt(timestamp):
+    return dt.datetime.fromtimestamp(timestamp, tz=timezone.utc)
+
+
+def parse_keys_json(resp_body):
+    keys = []
+    identity_scope = IdentityScope.UID2
+    for key in resp_body["keys"]:
+        keyset_id = None
+        if "keyset_id" in key:
+            keyset_id = key["keyset_id"]
+        key = EncryptionKey(key['id'],
+                            key.get('site_id', -1),
+                            _make_dt(key['created']),
+                            _make_dt(key['activates']),
+                            _make_dt(key['expires']),
+                            base64.b64decode(key['secret']),
+                            keyset_id)
+        keys.append(key)
+        if resp_body["identity_scope"] == "EUID":
```

Comment:
Why is this inside the keys loop?

Reply:
that's right, it doesn't need to be in the for loop. My indentation is incorrect